### PR TITLE
Simplify connectors

### PR DESCRIPTION
**Pull request by alexesprit**
Use new Connector API in Google Play Music and Pandora connectors.

### CODE REVIEW

File: connectors/googlemusic.js [hunk 1]
```diff
@@ -6,6 +6,17 @@ new class extends BaseConnector {
 
         this.name = 'Google Play Music';
         this.prefix = '/com/google';
+
+        this.artistsSelector = '#player-artist';
```
Comment (f1u77y) [inline connectors/googlemusic.js]: Isn't `artistSelector` equivalent to removed `get artist()`?

And this is element ID, so there shouldn't be more then one element.

Comment (alexesprit) [reply connectors/googlemusic.js]: That's a typo. I supposed to use `artistSelector`.

File: connectors/pandora.js [hunk 2]
```diff
@@ -6,9 +6,30 @@ new class extends BaseConnector {
...
+        this.artistSelector = '.Tuner__Audio__TrackDetail__artist';
+        this.titleSelector = '.Tuner__Audio__TrackDetail__title';
+
+        this.playButtonSelector = '.PlayButton';
+        this.nextButtonSelector = '.SkipButton';
+
         this.query('.region-bottomBar').then(elem => this.observe(elem));
     }
 
+    get currentTime() {
```
Comment (f1u77y) [inline connectors/pandora.js]: I've pushed c673337, now you could use default getters for this.

Comment (alexesprit) [reply connectors/pandora.js]: Updated according to the latest changes.

**Pull request merged**